posthog/hogql_queries/experiments/test/test_experiment_query_runner.py

```diff
@@ -1124,6 +1381,7 @@ def test_query_runner_with_internal_filters(self, name: str, filter: dict, expec
         ]
     )
     @snapshot_clickhouse_queries
+    @pytest.mark.skip(reason="Doesn't work with the new query runner")
```

Comment:
After 01ec4aa78e56ea72abcb3508763f2fe61f4f5f44, these tests fail with this:

```
E           clickhouse_driver.errors.ServerException: Code: 386.
E           DB::Exception: There is no supertype for types UUID, String because some of them are String/FixedString/Enum and some of them are not: JOIN  LEFT JOIN ... ON (exposure_data.aggregation_target = events_after_exposure.aggregation_target) AND (exposure_data.variant = events_after_exposure.variant) cannot infer common type in ON section for keys. Left key __table2.aggregation_target type UUID. Right key __table10.aggregation_target type String. Stack trace:
```

I think we can skip them for now and come back to them.

Reply:
Nevermind, I had time to fix this up e4abd8fb0e